### PR TITLE
Our impact page

### PR DESCRIPTION
**Pull request by SolivanLau**
**Styled Images**
-  created a **styled-img-container class** to create graphic behind
- should be global, but need time to refactor to global.  for now, it is in **impact.css** 

**Section Edits** 
- still need to adjust spacing between sections, but for now the components are responsive enough

**NEED TO Implement New Designer Graphics**
-  currently using old graphics, will replace later on

### CODE REVIEW

File: pages/impact.html [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+			</section>
+
+			<!-- Program Gallery -->
+			<section class="programs">
+				<div class="wrapper">
+					<!-- Program Icon gallery -->
+					<ul class="program-gallery">
+						<!-- Workshops and Programs -->
+						<li>
+							<img src="../assets/icons/pages/impact/workshops.svg" alt="" />
```
Comment (maxharrisnet) [inline pages/impact.html]: These icons are appearing really large for me. I think we need to set the height/width.

Screen Shot 2024-01-10 at 8 18 45 AM

File: pages/impact.html [hunk 2]
```diff
@@ -0,0 +1,399 @@
...
+						<h3>Responsible Stewardship</h3>
+						<p>
+							We understand that many technologies have not been used
+							responsibly and in inclusive ways, but we aim to improve that. We
+							believe in responsible stewardship with digital technologies.
+						</p>
+					</div>
+					<!-- styled img card  -->
+					<div class="styled-img-container style-right">
+						<img
```
Comment (maxharrisnet) [inline pages/impact.html]: Let's [lazy-load ](https://developer.mozilla.org/en-US/docs/Web/Performance/Lazy_loading) the images below the hero.

File: index.html [hunk 3]
```diff
@@ -360,7 +360,7 @@ <h4>Walking in Multiple Worlds</h4>
 					<!-- Company Column -->
 					<div class="company">
 						<!-- Footer Logo -->
-						<a href="" class="img-link">
+						<a href="./index.html" class="img-link">
```
Comment (maxharrisnet) [inline index.html]: A forward-slash works for getting to the root as well: `href="/"`

**Comment by maxharrisnet** [comment]
It looks like this branch was created from the `about-page` branch. This is causing the About changes to be included in this PR, which causes a couple of issues:
- Compared to the `development` branch, this PR contains code for the About page in addition to the intended Our Impact changes. 
- Changes in the `about-page` branch will fall out of sync with the changes here. 

Ideally, we would just have changes for one feature so that we can merge them in independently. Best practice is to start all new branches from the base branch (in this case, `development`). 

If you merge this on in first, the unfinished About page stuff in this branch would be included. We'll merge About in first, then you'll need to (locally) merge `development` into `our-impact-page` and resolve any merge conflicts (accept incoming changes, since `development` will have the latest for About). We can then merge `our-impact-page` `into development`.

I DON't think you need to bother starting this branch over or anything like that, just a heads-up on beast practices 😃 .

I hope that makes sense, but let me know if you have any questions or want to walk through this together.

File: css/impact.css [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+Section Styles (background, padding, etc.)
+===================================
+*/
+
+/* 
+===================================
+Technology & Stewardship Cards
+===================================
+*/
+.technology .wrapper,
```
Comment (maxharrisnet) [inline css/impact.css]: To keep this DRY, let's combine these `.tehcnology` and `.stewardship` classes into a generic class like `.section-2-col`  to be applied to any two-column section throughout the site (contact, our team hero, etc.).

The nested classes look good, so we can still use those (i.e. `.section-2-col .wrapper` ).

File: css/impact.css [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+.stewardship .wrapper {
+	display: flex;
+	gap: var(--spacing-xl);
+	padding: var(--spacing-xl);
+
+	width: var(--wrapper-width);
+	max-width: var(--wrapper-max-width);
+	margin: 0 auto;
+}
+.technology .styled-img-container,
```
Comment (maxharrisnet) [inline css/impact.css]:
```
.section-2-col  > div { ... }
```

File: css/impact.css [hunk 6]
```diff
@@ -0,0 +1,261 @@
...
+/* 
+===================================
+Programs Gallery
+===================================
+*/
+.programs {
+	position: relative;
+}
+
+/* overlapping background */
```
Comment (maxharrisnet) [inline css/impact.css:112]: I like how you set this up 👍🏾

File: impact.html [hunk 7]
```diff
@@ -0,0 +1,397 @@
...
+								alt="If These Lands Could Talk Logo"
+							/>
+						</a>
+						<!-- Navigation Menu -->
+						<div class="mobile-menu-container">
+							<!-- mobile menu button -->
+							<button
+								id="mobile-menu"
+								class="mobile-menu"
+								aria-label="toggle mobile menu"
```
Comment (maxharrisnet) [inline impact.html:44]: Thanks for using ARIA 🙌🏾

**Comment by maxharrisnet** [comment]
On mobile: spacing between sections needs some work

Screen Shot 2024-01-10 at 9 15 33 AM
Screen Shot 2024-01-10 at 9 15 26 AM

**Review by maxharrisnet — CHANGES_REQUESTED** [review]
This looks good. I made a comment about starting the PR from `development` rather than `about-page` branch. Not a big deal or anything you need to change right now. I just want to share that best practice with you for cleaner PRs and compartmentalization.

File: css/impact.css (hunk 4 above)
Comment (SolivanLau) [reply css/impact.css]: sounds good! will refactor and move those styles over to global.css

File: pages/impact.html (hunk 1 above)
Comment (SolivanLau) [reply pages/impact.html]: Sounds good, I am going to change out these icons for the newly approved icons from Pam and Tsista and get back to you on that

File: index.html (hunk 3 above)
Comment (SolivanLau) [reply index.html]: For sure, do you have a preference?

File: css/global.css [hunk 8]
```diff
@@ -815,6 +815,14 @@ Media Queries (overriding styles for different screen sizes)
 
 /* Tablet and smaller 768px */
 @media screen and (max-width: 768px) {
+	/* 
+	===================================
+	Global Section Adjustment
+	===================================
+	*/
+	section {
```
Comment (SolivanLau) [inline css/global.css:827]: using global section adjustments for spacing

File: css/impact.css [hunk 9]
```diff
@@ -230,8 +253,25 @@ Program Gallery
 	.program-gallery {
 		padding: var(--spacing-med) var(--spacing-sm);
 	}
+
+	/* gradient graphic adjustments */
+	.programs::before {
```
Comment (SolivanLau) [inline css/impact.css:264]: having to do a lot of experimenting w magic numbers - background looks more congruent for each breakpoint.

File: css/impact.css [hunk 10]
```diff
@@ -0,0 +1,277 @@
...
+}
+
+/* Tablet and smaller 768px */
+@media screen and (max-width: 768px) {
+	/* 
+===================================
+Hero 
+===================================
+*/
+	.hero {
```
Comment (maxharrisnet) [inline css/impact.css:237]: The spacing still looks off here. Let's center that text a little better.

Screen Shot 2024-01-10 at 11 32 33 AM

You probably need another class on the div that has the `.wrapper` class. It should add the following:

```
    display: flex;
    align-items: center;
    flex: 1 1 50%;
```
The `flex` property will make that box stretch to fill the 50% space. Override the gap from `.hero` to 0 as well. 

Screen Shot 2024-01-10 at 11 39 43 AM

File: index.html (hunk 3 above)
Comment (maxharrisnet) [reply index.html]: Let's use the `/` please

File: pages/about.html [hunk 11]
```diff
@@ -1,57 +0,0 @@
-<!DOCTYPE html>
```
Comment (maxharrisnet) [inline pages/about.html]: Let's make sure the file structure changes you made don't break. You may need to merge in the latest version of this file from `development`.

File: css/impact.css [hunk 12]
```diff
@@ -0,0 +1,277 @@
...
+}
+
+/* Mobile Only 470px */
+@media screen and (max-width: 470px) {
+	/* 
+===================================
+Mobile Fonts
+===================================
+*/
+	.technology h3,
```
Comment (maxharrisnet) [inline css/impact.css]: Can we replace these with `.section-2-col h3`? I think we can get rid of the content-specific classes altogether.

File: css/impact.css [hunk 13]
```diff
@@ -0,0 +1,277 @@
...
+		180deg,
+		#d3c53f 0%,
+		#d3c53f 26.04%,
+		#92a32d 58.85%,
+		#b37122 100%
+	);
+}
+
+/* layout */
+.program-gallery {
```
Comment (maxharrisnet) [inline css/impact.css:132]: This needs a border-radius

Screen Shot 2024-01-10 at 12 03 20 PM

File: css/about.css [hunk 14]
```diff
@@ -1,50 +1,115 @@
 /*
```
Comment (maxharrisnet) [inline css/about.css]: Careful not to break your about.css styles. I would say either checkout `about.css` and `about.html` from `development` or merge `development` in and accept incoming changes.

**Review by maxharrisnet — CHANGES_REQUESTED** [review]
Just a few more changes here - thanks @SolivanLau @SolivanLau!

File: css/impact.css (hunk 13 above)
Comment (SolivanLau) [reply css/impact.css:132]: added in!

File: pages/about.html (hunk 11 above)
Comment (SolivanLau) [reply pages/about.html]: merged in development - i do not think it broke any file structure

File: css/impact.css (hunk 12 above)
Comment (SolivanLau) [reply css/impact.css]: ooh yes - I missed that. Thanks for pointing it out!

Comment (SolivanLau) [reply css/impact.css]: I do need to close some padding on the stewardship content and the footer

File: css/impact.css (hunk 10 above)
Comment (SolivanLau) [reply css/impact.css:237]: added in - thank you for the deep dive!

File: css/about.css (hunk 14 above)
Comment (SolivanLau) [reply css/about.css]: thank you! merged development into impact branch and have no conflicts

**Review by maxharrisnet — APPROVED** [review]

**Pull request merged**